front50-core/src/main/java/com/netflix/spinnaker/front50/echo/Event.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.front50.echo;
+
+public abstract class Event {}
```

Comment:
Do we need?

Reply:
I lifted this from the -nflx code.  I think the point was to have a supertype for `EchoService`'s `String postEvent(@Body Event event);` instead of typing `postEvent` as, for example `PluginEvent` or `Object`.   This could probably be a marker interface instead.